dynamodb/README.md

```diff
@@ -0,0 +1,134 @@
+# HarbourBridge: Turnkey DynamoDB-to-Spanner Evaluation
+
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation,
+using data from an existing PostgreSQL or MySQL or DynamoDB database. This
+README provides details of the tool's DynamoDB capabilities. For general
+HarbourBridge information see this [README](https://github.com/cloudspannerecosystem/harbourbridge#harbourbridge-turnkey-spanner-evaluation).
+
+## Example DynamoDB Usage
+
+The following examples assume `harbourbridge` has been added to your PATH
+environment variable.
+
+HarbourBridge can be run directly on a DynamoDB database (via the aws go sdk
```

Comment:
This sentence doesn't add too much -- it was originally there to explain the dump vs direction functionality, which isn't relevant to DynamoDB.

Suggest either just deleting it.
Or may preserving the comment in parenthesis (since that is kind of useful).
Maybe add a short sentence to line 18: "HarbourBridge accesses your DynamoDB database via the aws go sdk package."